alerts.md

```diff
@@ -165,6 +169,93 @@ Example Helm `values.yaml`:
 
 ----
 
+
+### Type: Asset Budget
+
+Define asset budgets and alert on asset that overruns the threshold set.
+
+*Required parameters:*
+* `type: assetBudget`
+* `threshold: <amount>` -- cost threshold in configured currency units
+* `aggregation: <agg-parameter>` -- configurable, accepts all aggregations supported by the [asset API](https://github.com/kubecost/docs/blob/2ea9021e8530369d53184ea5382b2e4c080bb426/assets.md)
+* `filter: <value>(,<value>,...)` -- configurable, accepts any 1 or more filter values which are comma seperated values
+* `window: <N>d` or `<M>h` -- configurable, (1 ≤ N ≤ 7, 1 ≤ M ≤ 24)
+
+*Valid Aggregation Parameters*: 
+* `category`
+* `cluster`
+* `service`
+* `type`
+* `provider`
+* `account`
+* `providerid`
+* `label` requires the following format: `label:<label_name>=<label_value>`
+
+Example Helm `values.yaml`:
+
+```
+# Daily namespace budget alert on type `Node,Loadbalancer`
+- type: assetBudget
+  threshold: 50
+  window: daily # or 1d
+  aggregation: type
+  filter: Node,LoadBalancer
+# 3d cluster budget alert on cluster with no filter set
+- type: assetBudget
+  threshold: 100
+  window: 3d
+  aggregation: cluster
+  filter: ''
+```
+
+----
+
+### Type: Cloud Report
+
+Sends a recurring email and/or Slack alert with a asset summary report of cost and efficiency metrics. 
+
+*Required parameters:*
+* `type: cloudReport`
+* `aggregation: <aggregation>` 
+* `filter: ''`
+* `window: <N>d`
+
+*Valid Window Parameters*:
+* `<N>d` where `N in [1, 7)` for every N days
+* `7d` or `weekly` for 0:00:00 UTC every Monday
+* `30d` or `monthly` for 0:00:00 UTC on the first day of the month.
+
+*Valid Aggregation Parameters*: 
+* `category`
+* `cluster`
+* `service`
+* `type`
+* `provider`
+* `account`
+* `providerid`
+* `label` requires the following format: `label:<label_name>=<label_value>`
+
+Example Helm `values.yaml`:
```

Comment:
I think values.yaml should be italicized rather than formatted as inline code.

Reply:
Changed all references of values.yaml to italicize. A Question why do you as a technical writer prefer italicizing over inline in this instance?

Reply:
So I may need a bit more context as to what values.yaml is, as it shows up all over the documentation, but if it is referring to a specific file with a name and format, I would italicize it the same way I would with interactable text elements. If we refer vaguely to a .yaml file (or any file format like .jpg, .png, etc), no italics needed. I just don't think we should use '' because I would only use that for code examples.

Reply:
Thanks, Was just asking for future documentation. I think Italicizing makes sense over backticks here. Values.Yaml in simple words is a source of truth for all kubernetes helm deployment.

Reply:
Okay. If you don't have a link to our internal Style guide I can send that to you.